app/main.py

```diff
@@ -1,3 +1,7 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    counter = 0
+    for char in phrase:
+        if char.lower() == letter.lower():
+            counter += 1
+
```

Comment:
There is an indentation issue here. The line should be indented to align with the `for` loop block. This will ensure that the `return` statement is executed after the loop completes.